app/assets/stylesheets/active_admin/_header.scss

```diff
@@ -90,50 +77,41 @@
       }
 
       &.has_nested:hover > a {
-        @include rounded-top(10px);
         border-bottom: 5px solid $hover-menu-item-background;
         background: $hover-menu-item-background image-url('active_admin/nested_menu_arrow_dark.gif') no-repeat calc(100% - 7px) 50%;
         z-index: 1020;
       }
 
-
       /* Hover on li, display the ul */
-      &:hover ul { display: block;}
+      &:hover ul {
+        display: block;
+      }
+
       /* Drop down menus */
       ul {
         background: $hover-menu-item-background;
-        @include rounded-all(0,10px,10px,10px);
-        @include shadow(0, 1px, 3px, #444);
-        position: absolute;
-        width: 120%;
-        min-width: 175px;
-        max-width: calc(100% + 20px);
-        margin-top: 5px;
-        float: left;
+        border-radius: 0 4px 4px 4px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `0 4px 4px` instead of `0 4px 4px 4px`